opty/direct_collocation.py

```diff
@@ -727,6 +729,77 @@ def __init__(self, equations_of_motion, state_symbols,
             If True, STDOUT and STDERR of the Cython compilation call will be
             shown.
 
+        Attributes
+        ----------
+        Some of the attributes are explained in more detail under Parameters above.
+
+        eom: sympy.Matrix
+            The equations of motion used.
+        state_symbols : tuple
```

Comment:
```suggestion
        state_symbols : 2*n-tuple
```

You can indicate the length of the tuple like so. Or 3-tuple, n-tuple, etc.

Reply:
> You can indicate the length of the tuple like so. Or 3-tuple, n-tuple, etc.

I will so this.
Rendering o.k.?

Reply:
I don't  know, you will have to work out the size of any iterables.

Reply:
> I don't know, you will have to work out the size of any iterables.

I am not sure I understand:
if, by definition, n is the number of state variables, the would state_symbols not have to be an n-tuple and eom would have to be an (n,1) sympy Matrix?

Reply:
I think so, but I don't remember off the top of my head.

Reply:
> I think so, but I don't remember off the top of my head.

These are the definitions in the docstring of the class, so I take them.